frontend/src/lib/components/Lettermark/Lettermark.tsx

```diff
@@ -3,30 +3,41 @@ import React from 'react'
 import './Lettermark.scss'
 
 export enum LettermarkColor {
-    Default = 'default',
-    Gray = 'gray',
+    Default = 1,
+    Gray = 8,
```

Comment:
This isn't really gray, just one of the colored variants. This change doesn't look great in funnels for instance:

![Zrzut ekranu 2022-08-8 o 13 12 49](https://user-images.githubusercontent.com/4550621/183405418-c0bb0ebb-f126-4bdc-b63c-21c571a4800e.png)

Can we keep the existing gray as a variant?

Reply:
ah good catch. Sure I'll add it in as a unique variant perhaps...